### DISCUSSION TITLE
Off-Topic Question: How did you implemented TinyMCE in Svelte?

### DISCUSSION THREAD

**Discussion by MuslemRahimi** [pocketbase/pocketbase#4137] [Q&A]
I am still struggling to self-host tinymce but there is no good example how to do it in svelte.

Can you help me out me please. I couldn't find the code snippet in your repo

**Comment by jonshipman** [comment]
Svelte !== Pocketbase. 

That said, you should be able to (at the very least) copy the tinymce JS to the static folder, link it in app.html, and use it regularly inside svelte files (this is in kit). If you're not using Sveltekit, you'd need to consult the compiler you're using.

**Comment by ganigeorgiev** [comment]
You can use https://github.com/tinymce/tinymce-svelte and point the `scriptSrc` to a static TinyMCE bundle JS file.

We used to use the above initially but because of some peculiarities around the event loading, we now use a slightly modified version of the above component - [TinyMCE.svelte](https://github.com/pocketbase/pocketbase/blob/master/ui/src/components/base/TinyMCE.svelte).

**Comment by MuslemRahimi** [comment]
Thank you both for you answers. I couldn't found either a dark theme such as oxide-theme. Anyone knows where I can find such a thing?
